example-config.yml

```diff
@@ -1,15 +1,17 @@
 api:
-  host: http://trimet.dev.conveyal.com
+  host: https://trimet-otp.ibi-transit.com
```

Comment:
Let's make this point to localhost instead since this is merely an example and we don't want to tie use to any quasi-production server.